src/detect-engine-address.c

```diff
@@ -137,23 +137,6 @@ DetectAddress *DetectAddressCopy(DetectAddress *orig)
     return ag;
 }
 
-#ifdef DEBUG
-/**
- * \brief Prints the address data information for all the DetectAddress
- *        instances in the DetectAddress list sent as the argument.
- *
- * \param head Pointer to a list of DetectAddress instances.
- */
-void DetectAddressPrintList(DetectAddress *head)
-{
-    SCLogInfo("list:");
-    for (DetectAddress *cur = head; cur != NULL; cur = cur->next) {
-        DetectAddressPrint(cur);
-    }
-    SCLogInfo("endlist");
-}
-#endif
-
```

Comment:
This one stands out, is it useful when debugging?

Reply:
It is unused... there is no debug logging calling this. So I would say not so useful..?